src/main/java/com/openlattice/data/DataGraphIds.java

```diff
@@ -23,23 +23,24 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.google.common.collect.ListMultimap;
 import com.openlattice.client.serialization.SerializationConstants;
 
+import java.util.List;
+import java.util.Map;
 import java.util.Objects;
 import java.util.UUID;
 
 /**
  * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
  */
 public class DataGraphIds {
```

Comment:
Just a note to multimap -> map changes: this class should be fine looking at the old/new implementation.